packages/runtime/src/schema.ts

```diff
@@ -199,6 +199,17 @@ const UpdateOptionsInput = new GraphQLInputObjectType({
   },
 })
 
+const EnableIndexingOptionsInput = new GraphQLInputObjectType({
+  name: 'EnableIndexingOptionsInput',
+  fields: {
+    streamID: {
+      type: GraphQLString,
+      description: 'Reference a specific and unique event stream',
+    },
+    shouldIndexField,
+  },
+})
+
```

Comment:
I don't think this is needed, the top-level fields of the mutation input should provide all the arguments
```suggestion
```